Rakefile

```diff
@@ -35,7 +35,7 @@ task :validate_environment do
   end
 
   unless ENV.include?('TF_VAR_account_id')
-    warn 'Please set the "TF_VAR_account_id" environment variable.'
+    warn 'Please set the "TF_VAR_account_id" environment variable to your AWS access ID.'
```

Comment:
Technically yes but for clarity, I think it should read something like `Please set the "TF_VAR_account_id" environment variable to the name of the aws account` i.e. "govuk-infrastructure-integration".  Nice one addressing the enigmatic warning message though.